src/components/ReturningVoterMessage/ReturningVoterMessage.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { Modal } from "semantic-ui-react";
+
+class ReturningVoterMessage extends Component {
+  
+  state = {
+    modalOpen: true,
+  };
+
+  render() {
+
+    return (
+      <div>
+        <Modal size="mini" open={this.state.modalOpen}>
+          <Modal.Content>
+            <h4>
+              You have already voted and may not vote twice.
```

Comment:
Just a thought on wording: "Our records indicate you have already voted." ?

Reply:
that sounds good. was struggling with the copy, so thanks for catching that.